README.md

```diff
@@ -8,6 +8,10 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/KohlsTechnology/eunomia)](https://goreportcard.com/report/github.com/KohlsTechnology/eunomia)
 [![codecov](https://codecov.io/gh/KohlsTechnology/eunomia/branch/master/graph/badge.svg)](https://codecov.io/gh/KohlsTechnology/eunomia)
 
+## Getting started
+
+If you want to deploy Eunomia from scratch to a local Minikube cluster, begin with the [Getting started](./GETTING_STARTED.md) document. It will guide your through the setup step-by-step.
+
 ## Who is Eunomia
 
 According to Wikipedia:
```

Comment:
Line 52, the first reference of "CR" should have the full work after i.e. "CR (Custom Resource)". Same with line 58 first reference of "CRD" should have definition (i.e. "CRD (Custom Resource Definition)")